src/_common/realm/AppRealmFullCard.vue

```diff
@@ -115,9 +122,11 @@ async function onClick(event: Event) {
 		/>
 
 		<AppResponsiveDimensions :ratio="REALM_CARD_RATIO">
-			<AppMediaItemBackdrop v-if="mediaItem" :media-item="mediaItem">
-				<AppImgResponsive class="-cover-img" :src="mediaItem.mediaserver_url" alt="" />
-			</AppMediaItemBackdrop>
+			<RouterLink v-if="to" :to="to">
```

Comment:
i didnt realize you could use RouterLink in `component :is`, thats neat

Reply:
can we use "template" instead so we don't unnecessarily nest it?

Reply:
doesnt seem to work. dang, can't do a "vanishing component" either because each component must have one root element, and if a component vanishes and has multiple children it'll end up rendering multiple root components either. guess its not worth it.